src/doc/tarpl/ownership.md

```diff
@@ -0,0 +1,66 @@
+% Ownership and Lifetimes
+
+Ownership is the breakout feature of Rust. It allows Rust to be completely
+memory-safe and efficient, while avoiding garbage collection. Before getting
+into the ownership system in detail, we will consider the motivation of this
+design.
+
+We will assume that you accept that garbage collection is not always an optimal
+solution, and that it is desirable to manually manage memory to some extent.
+If you do not accept this, might I interest you in a different language?
+
+Regardless of your feelings on GC, it is pretty clearly a *massive* boon to
+making code safe. You never have to worry about things going away *too soon*
+(although whether you still *wanted* to be pointing at that thing is a different
+issue...). This is a pervasive problem that C and C++ need to deal with.
```

Comment:
C and C++ *programs* need to deal with the problem.  (The way you phrase the sentence and given how the wordw "Rust" is used as a subject in other contexts, it sounds like the languages themselves need to one day attempt to deal with them, which seems absurd.)